categories/basis.tex

```diff
@@ -511,7 +511,7 @@
 % engl. "Prancing"
 \whitecard{Basis}{Teambildungs- maßnahmen.}%
 % original: -übungen
-\whitecard{Basis}{Teenager- schwangerschaft.}%
+\whitecard{Basis}{Teenager-schwanger-schaft.}%
```

Comment:
Why did you break `Schwangerschaft` into `Schwanger-schaft` here?

Reply:
@Prior99:
here is the render for 
```tex
\whitecard{Basis}{Teenager-schwangerschaft.}%
```
![no break](https://user-images.githubusercontent.com/3804909/38930621-d4a4bd90-4310-11e8-8bd1-cc613eec5b1e.png)  
It also leads to a warning:
```log
{/usr/local/texlive/2017/texmf-var/fonts/map/pdftex/updmap/pdftex.map}] [2] [3]
[4]
Overfull \hbox (4.9202pt too wide) in paragraph at lines 42--42
\T1/fav/m/n/17.28 schwangerschaft.  
 []
```

in contrast, here the render for
```tex
\whitecard{Basis}{Teenager-schwanger-schaft.}%
```
![with break](https://user-images.githubusercontent.com/3804909/38930657-eb784bcc-4310-11e8-8a81-14fd417f90c8.png)